src/app/app.component.html

```diff
@@ -1,5 +1,9 @@
 <general-info></general-info>
+<br><br>
```

Comment:
This should probably be accomplished with paddings and margins

Reply:
Done